envi/config.py

```diff
@@ -180,10 +180,16 @@ def parseConfigOption(self, optstr):
             valstr = str(int(valstr, 16))
 
         if not (valstr.startswith('"') and valstr.endswith('"')):
-            try:
-                int(valstr)
-            except:
-                valstr = '"' + valstr + '"'
+            if valstr.lower() == 'true':
```

Comment:
If we're going to break from requiring the option arguments to be legit json ( which it seems like we kinda have already via 0x prefix ) perhaps we should just move to ast.literal_eval and require string args to be quoted?  Never been a huge fan of the bastardized syntax, but cli users expect a level of simplicity to the options as well.. :(